schemas/mod-source-record-manager/DeleteJobExecutionsDto.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Request to initialize JobExecution entities",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "ids": {
+      "description": "Information about files to upload",
```

Comment:
Hello @SatishkumarPurswani, could you please update the description according to DTO purpose. 
Something like this: "Id of JobExecution entities to be deleted".